examples/characters.jl

```diff
@@ -0,0 +1,33 @@
+include("../src/characters.jl")
+
+# Build group ring for the permutation group over 4 elements.
+N = 4
+G = PermGroup(N)
+RG = GroupRing(G)
+
+# Conjugacy classes as all-one elements of GroupRing
+# WTH does all-one elements mean?
```

Comment:
treating elements of RG as function G→R (coefficient ring) it's a characteristic function of a conjugacy class (understood as a simple set). Or a 0-1 coefficients polynomial on a given set of monomials (here: the conjugacy class)